PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!--
+Thanks for being here!
+Review these resources before making a contribution:
+  - Code of Conduct: https://github.com/target/markdown-inject/blob/main/CODE_OF_CONDUCT.md
+  - Contribution guide: https://github.com/target/markdown-inject/blob/main/CONTRIBUTING.md
+-->
+
+closes #<issue number>
+
+# Changes
```

Comment:
I purposefully used `Changes` over my usual `Features` because not all changes are features, and it feels weird to write a bug fix or documentation change into a PR body as a "feature".